Apps/GatewayApi/src/Models/DependentRecord.cs

```diff
@@ -0,0 +1,41 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+
+namespace HealthGateway.GatewayApi.Models
+{
+    using System;
+
+    /// <summary>
+    /// Represents a dependent record.
+    /// </summary>
+    public class DependentRecord
```

Comment:
I'd prefer to move this class (as a record) in the `GetDependentResponse.cs` as it doesn't merit its own file, but this may be a team discussion.

Reply:
Team discussion.

Reply:
Now removed.